data/json/monsters.json

```diff
@@ -4644,6 +4644,52 @@
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_thorny",
+    "type": "MONSTER",
+    "name": "thorny shambler",
+    "description": "A shambling undead humanoid, body overgrown with a tangled morass of otherwordly plantlife.  Hunched over and dragging its feet as it moves, the bramble utterly enveloping its upper half twitches and moves with a life of its own, tendrils tipped with a paralytic sting.",
```

Comment:
I'd suggest using "vegetation" instead of "plantlife" here, since "plantlife" isn't a word and it's not exactly alive.